test/wrapper.test.ts

```diff
@@ -202,64 +191,22 @@ describe('Tests Native Wrapper', () => {
 
       await NATIVE.sendEnvelope(env);
 
-      expect(RNSentry.captureEnvelope).toBeCalledWith({
-        header: {
-          event_id: event.event_id,
-          sent_at: '123',
-        },
-        payload: {
-          ...event,
-          message: event.message,
-        },
-      });
-    });
-    test('serializes class instances on iOS', async () => {
-      class TestInstance {
-        value: number = 0;
-        method = () => null;
-      }
-
-      const event = {
-        event_id: 'event0',
-        message: 'test',
-        sdk: {
-          name: 'test-sdk-name',
-          version: '2.1.3',
-        },
-        instance: new TestInstance(),
-      };
-
-      const env = createEnvelope<EventEnvelope>({ event_id: event.event_id, sent_at: '123' }, [
-        [{ type: 'event' }, event] as EventItem,
-      ]);
-
-      await NATIVE.sendEnvelope(env);
-
-      expect(RNSentry.captureEnvelope).toBeCalledWith({
-        header: {
-          event_id: event.event_id,
-          sent_at: '123'
-        },
-        payload: {
-          ...event,
-          message: event.message,
-          instance: {
-            value: 0,
-          },
-        },
-      });
+      expect(RNSentry.captureEnvelope).toBeCalledWith(
+        utf8ToBytes('{"event_id":"event0","sent_at":"123"}\n'
+          + '{"type":"event","content_type":"application/json","length":87}\n'
+          + '{"event_id":"event0","message":"test","sdk":{"name":"test-sdk-name","version":"2.1.3"}}\n'
```

Comment:
Can we assign the value to a string and do proper indentation to make the tests readable?

Reply:
I was also thinking about that, but when assigned to template string we cant keep the indentation. So it also looks a bit strange.